gnucash/report/standard-reports/transaction.scm

```diff
@@ -555,6 +574,16 @@ tags within description, notes or memo. ")
     'non-void-only
     (keylist->vectorlist show-void-list)))
 
+  (gnc:register-trep-option
+   (gnc:make-multichoice-option
+    pagename-filter optname-closing-transactions
+    "l" (_ "By default most users should not include closing \
+transactions in a transaction report. Closing transactions are \
+transfers from INCOME and EXPENSE accounts to equity, and must usually \
+be excluded from periodic reporting.")
+    'exclude-closing
+    (keylist->vectorlist show-closing-list)))
+
```

Comment:
This particular commit adds plenty of new translatable strings. Our translators won't be happy if we do that on the maint branch. There was a previous commit that introduced one new string which I would let pass.  I'm counting 8 here.